Projects/Feature/SigninFeature/Project.swift

```diff
@@ -0,0 +1,20 @@
+import ProjectDescription
+import ProjectDescriptionHelpers
+import DependencyPlugin
+
+let project = Project.makeModule(
+    name: ModulePaths.Feature.SigninFeature.rawValue,
+    product: .staticFramework,
```

Comment:
```suggestion
    product: .staticLibrary,
```

번들 메인쓰니 이거도 staticLibrary로 바꾸죠